ci/sync-mirrors.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+set -e
+set -o pipefail
+
+BUILD_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
+source "${BUILD_DIR}/ci/common/common.sh"
+source "${BUILD_DIR}/ci/common/dependencies.sh"
+
+WORK_DIR=${WORK_DIR:-${BUILD_DIR}/build}
+MIRROR_USER=${MIRROR_USER:-neovim}
+MIRROR_BRANCH=${MIRROR_BRANCH:-master}
+
+sync() {
```

Comment:
sync (1) is actually a common shell command. Just for good practice, I would suggest a more specific name.